.changes/unreleased/Breaking Changes-20230718-130019.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Breaking Changes
+body: Dropped support for Python 3.7
```

Comment:
nit: `Drop support for Python 3.7` same for the commit message

Reply:
done